harness/determined/estimator/_estimator_trial.py

```diff
@@ -534,6 +576,8 @@ def _init_model(self) -> None:
         self._init_val_hooks()
         self._init_paths()
 
+        self.train_hooks.append(RNGStateHook(self.estimator_dir))
```

Comment:
blocking: this probably should be embedded into `DeterminedControlHook` rather than stand-alone.

Reply:
Yeah that makes sense. I had to shuffle a couple of things because it depends on parts of init_paths() which depends on ini_train_hooks. Some harness tests failed locally but I'm not certain they're related - I've had some flakiness running them on my machine. Let's see how Circle does...